test/common/TestBed.cpp

```diff
@@ -229,8 +229,12 @@ namespace RcclUnitTesting
     for (int childId = 0; childId < this->numActiveChildren; ++childId)
     {
       PIPE_WRITE(childId, cmd);
+      int tempCurrentRanks = currentRanks.size();
```

Comment:
This shouldn't be necessary - sending the single rank value (similar to other functions in TestBed) should suffice, with rank = -1 once again representing all ranks.